src/Microsoft.Diagnostics.Monitoring.WebApi/Stacks/NameFormatter.cs

```diff
@@ -24,6 +24,8 @@ internal sealed class NameFormatter
         private const char GenericStart = '[';
         private const char GenericSeparator = ',';
         private const char GenericEnd = ']';
+        private const char OpenParentheses = '(';
```

Comment:
Name these for what they are used, not what they literally are e.g. `MethodParametersStart` and `MethodParametersEnd`